modules/ting/src/TingLogInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @file
+ * Log interface.
```

Comment:
I would much prefer to see something like [`psr3_watchdog`](https://www.drupal.org/project/psr3_watchdog) used here. I know it is mentioned in the commit message but I fear it may be too late now 😞 .